_app/plugins/gatsby-remark-tamburetei/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "gatsby-remark-tamburetei",
+  "version": "0.0.1",
+  "private": true,
+  "license": "UNLICENSED",
```

Pode botar MIT aqui 😄 
```suggestion
  "license": "MIT",
```

---

top!